zimbra_all/opt/push-logs-to-logstash/02-output.conf

```diff
@@ -0,0 +1,35 @@
+# This line sends all lines to defined IP address at port 10514,
+# using the "json-template" format template
+#
+####### server logtash config listen port 10514/udp like: /etc/logstash/conf.d/logstash.conf
+# # This input block will listen on port 10514 for logs to come in.
+# # host should be an IP on the Logstash server.
+# # codec => "json" indicates that we expect the lines we're receiving to be in JSON format
+# # type => "rsyslog" is an optional identifier to help identify messaging streams in the pipeline.
+
+# input {
+#   udp {
+#     host => "logstash_ip"
+#     port => 10514
+#     codec => "json"
+#     type => "rsyslog"
+#   }
+# }
+
```

Comment:
What does this block mean?

Reply:
It is used as a hint to configure logstash to listen to port 10514/udp.